src/main/java/com/theelixrlabs/healthcare/service/PatientService.java

```diff
@@ -0,0 +1,45 @@
+package com.theelixrlabs.healthcare.service;
+
+import com.theelixrlabs.healthcare.constants.PatientConstants;
+import com.theelixrlabs.healthcare.dao.PatientRepository;
+import com.theelixrlabs.healthcare.dto.PatientDTO;
+import com.theelixrlabs.healthcare.model.PatientModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service class for managing patient-related operations.
+ * Validations on the input parameters are done in this class.
+ */
+@Service
+public class PatientService {
+    @Autowired
+    private PatientRepository patientRepository;
+
+    /**
+     * Creates a new patient based on the provided PatientDTO.
+     *
+     * @param patientDTO    The data transfer object containing patient information.
+     * @return The PatientDTO of the newly created patient, with ID populated.
+     * @throws RuntimeException If the aadhaar number already exists in the database.
+     */
+    public PatientDTO createPatient(PatientDTO patientDTO) throws RuntimeException {
```

Comment:
I feel addPatient or addPatientDetails makes more sense than createPatient. What do you think?